user.py

```diff
@@ -15,34 +15,38 @@ def __init__(self, username=None, email=None, password=None, cpassword=None):
         self.email = email
         self.password = password
         self.cpassword = cpassword
-
     def register(self, email, username, password, cpassword):
         """defining method to create account"""
         if username != '' and email != '' and password != '':
```

Comment:
Note that if an `if` condition is met, an action is done. If not, the `if` statement is ignored. Your logic is correct but your `return` statements should be reorganised